README.md

```diff
@@ -128,4 +128,7 @@ CLIENT=[Your Connection String Here]
 
 2. Add CLIENT secret to Github
 3. Add CLIENT and some new DB eg 'PRODDB' as DBNAME as Heroku config variables
-```
\ No newline at end of file
+```
+
+## Linking to OAuth
+To-Do App is using GitHub for linking to default OAuth provider to manage user access. Users will now be redirected to github to sign and access the application. You will need to register your application with your chosen Auth provider and populate your .env file with the enviroment variables found in the .env.template with the values taken from Github OAuth. Users with "read" access will not be allowed to add, update or delete the tasks in the app.
```

Comment:
Minor, but as you say we're using GitHub, and choosing an alternative Auth Provider won't _just work_ - we'd have to change the code to allow for that!

Reply:
Done